server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMailboxesMethodTest.java

```diff
@@ -96,8 +166,262 @@ public void setMailboxesShouldCreateMailbox() throws Exception {
         .when()
             .post("/jmap")
         .then()
+            .log().ifValidationFails()
```

Comment:
It's already present in a lot of tests when checking for the returned JSON.
It only logs when one of the assertions fails, so we don't have so much logs.